scalalib/worker/src/mill/scalalib/worker/ZincWorkerImpl.scala

```diff
@@ -534,7 +534,10 @@ class ZincWorkerImpl(
       stampReader = Stamps.timeWrapBinaryStamps(converter)
     )
 
+    val scalaColorProp = "scala.color"
+    val previousScalaColor = sys.props.get(scalaColorProp).getOrElse("auto")
```

Comment:
Inspired by sbt: https://github.com/sbt/sbt/blob/cfe0fc8e2ee3c551389fc24d836b04764e9a2e2b/main-actions/src/main/scala/sbt/Console.scala#L68

Reply:
This means, we will later "reset" it to a new value "auto" instead of the original value, in case it was not defined. I'd rather reset to the original, which means we need to unset when it was unset before.

Reply:
I agree with you.